core/domain/wipeout_service.py

```diff
@@ -69,3 +68,65 @@ def pre_delete_user(user_id):
         explorations_to_be_deleted_ids,
         collections_to_be_deleted_ids,
     )
+
+
+def delete_user(pending_deletion_model):
+    """Delete all the models for user specified in pending_deletion_model.
+
+    Args:
+        pending_deletion_model: PendingDeletionRequestModel.
+    """
+    delete_user_models(pending_deletion_model.id)
+    pending_deletion_model.deletion_complete = True
+    pending_deletion_model.put()
+
+
+def verify_user_deleted(pending_deletion_model):
+    """Verify that all the models for user specified in pending_deletion_model
+    are deleted.
+
+    Args:
+        pending_deletion_model: PendingDeletionRequestModel.
+
+    Returns:
+        bool. True if all the models were correctly deleted, False otherwise.
+    """
+    if verify_user_models_deleted(pending_deletion_model.id):
```

Comment:
Maybe make _verify_user_models_deleted private to this module? (Test it through the public API exposed by this function.)

Otherwise you're exposing two public functions with names that are too similar.

Reply:
Done.